python/PyQt6/gui/auto_additions/qgsrecentcoordinatereferencesystemsmodel.py

```diff
@@ -1,15 +1,2 @@
 # The following has been generated automatically from src/gui/proj/qgsrecentcoordinatereferencesystemsmodel.h
-QgsRecentCoordinateReferenceSystemsModel.RoleNodeType = QgsRecentCoordinateReferenceSystemsModel.Roles.RoleNodeType
```

Comment:
Note that these were incorrectly included in an earlier PR during this development cycle!